### DISCUSSION TITLE
Modifying Schema with same version causes problem when tables grow more than 50 or so

### DISCUSSION THREAD

**Discussion by abduljabbar021** [dexie/Dexie.js#1941] [Q&A]
I think there is some issue when updating schema with same number of version but more than 50 schema. It isn't creating new tables. Table is not shown in indexedDB in Console as well even though I could see that my db connection has this table

    db.version(1).stores({tbl1:'++id, aid', tbl2:'++id,bid', tbl55:'++id,cid'});        //same version with updated Schema including previous tables
    db.table('tbl55');                           //It returns table information with its schema etc but when trying to add rows in it, it throws error
    db['tbl55'].bulkPut(rows);             //It throws error as NotFoundError, 'tbl55' is not a known object store

But when trying to update version as well with new schema, it works good

    db.version(2).stores({tbl55:'++id,cid'});      //new version
    db.table('tbl55');                                         //It returns table information with its schema
    db['tbl55'].bulkPut(rows);                           //It works well without any issue.

Here is the code

    dbManager.updateSchema({tbl55:'++id, cid'}); //Adding new table tbl55

    //Called to add new schema to database
    async updateSchema(schema){ 
        let schemas = localStorage.getItem('dbSchemas');
        schemas = (schemas ? JSON.parse(schemas) : {});
        schemas = Object.assign(schemas, schema);
        localStorage.setItem('dbSchemas', JSON.stringify(schemas));
        this.db.close();
        this.db.version(1).stores(schemas); //Using Fixed number of version but updated schema, it causes problem
        await this.db.open();
    }

    //Called on page refresh so, it can load with up to date schema
    loadUsersVersions(){  
        let schemas = localStorage.getItem('dbSchemas');
        if(schemas) this.db.version(1).stores(JSON.parse(schemas));
    }

Above code run into issue

**Comment by dfahlander** [comment]
Thanks! Please provide a full repro in jsfiddle or similar where I can open a link the reproduces the issue.
